src/types/annotator.js

```diff
@@ -186,9 +188,42 @@
  */
 
 /**
- * Specifies a content partner/provider to show branding for.
+ * Content provider logo details.
  *
- * @typedef {'jstor'} ContentPartner
+ * @typedef ContentInfoLogo
+ * @prop {string} logo
+ * @prop {string} title
+ * @prop {string} link
+ */
+
+/**
+ * Metadata for the current document, for display in the content info banner.
```

Comment:
Note from Lyza. We need to add an item type here in order to generate appropriate labels for the previous/next-item links. eg. "Previous chapter" or "Previous article". See https://hypothes-is.slack.com/archives/C4K6M7P5E/p1658414917099319?thread_ts=1658412471.267439&cid=C4K6M7P5E

Reply:
There is an ongoing discussion in that thread about whether to instead put labels for the links in the config (eg. `{ url: "...", title: "Previous chapter }`) so the client doesn't have to know about all the different item types that a content provider might have.